recipes/arrow/all/conanfile.py

```diff
@@ -376,7 +376,7 @@ def requirements(self):
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version],
-                  destination=self.source_folder, strip_root=True)
+                  filename=f"apache-arrow-{self.version}.tar.gz", destination=self.source_folder, strip_root=True)
```

Comment:
why?

Reply:
Without this, we get the following error:

```text
ERROR: arrow/10.0.0: Error in source() method, line 378
	get(self, **self.conan_data["sources"][self.version],
	ConanException: Cannot deduce file name from the url: 'https://www.apache.org/dyn/closer.lua/arrow/arrow-10.0.0/apache-arrow-10.0.0.tar.gz?action=download'. Use 'filename' parameter.
```

Reply:
I guess is because the parameter `?action=download`. Is it really necessary?